src/components/widgets/multi-select/multi-select.tsx

```diff
@@ -69,7 +75,15 @@ export const MultiSelect = ({
                         textAlign='left'
                         variant='outline'
                         colorScheme='black'
-                        borderColor={isOpen || selected.length ? 'lime.300' : 'blackAlpha.200'}
+                        px={maxVisibleTags && 3}
+                        borderWidth={isInvalid ? '2px' : undefined}
+                        borderColor={
+                            isInvalid
+                                ? 'red.500'
+                                : isOpen || selected.length
+                                  ? 'lime.300'
+                                  : 'blackAlpha.200'
+                        }
                         isDisabled={isDisabled}
                     >
                         {selected.length === 0 || isDisabled ? (
```

Comment:
```suggestion
                        {selected.length || isDisabled ? (
```

вроде как должно работать и таким образом